bazel_versions.bzl

```diff
@@ -1,10 +1,10 @@
 """Tested/Supported Bazel Versions"""
 
-CURRENT_BAZEL_VERSION = "5.1.0"
+CURRENT_BAZEL_VERSION = "5.1.1"
```

Comment:
curious, is this upgrade required to make the test runner work?

Reply:
No. I should have put that into a separate PR. The fix was upgrading to the latest rules_spm which has the latest rules_swift.